docs/cloud/azureai/tracing/index.md

```diff
@@ -0,0 +1,84 @@
+# Tracing from local to cloud
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../../../how-to-guides/faq.md#stable-vs-experimental).
+:::
+
+Prompt flow [tracing feature](../../../how-to-guides/tracing/index.md) enables users to trace LLM calls, functions and even LLM frameworks. Besides, with Azure extension installed, prompt flow can also log traces to an Azure ML workspace or Azure AI project, which makes it possible to share traces with your team members.
+
+## Installing the package
+
+```console
+pip install "promptflow[azure]>=1.11.0"
+```
+
+## Set cloud destination
+
+To log traces to cloud, first of all, you need an [Azure ML workspace](https://learn.microsoft.com/en-us/azure/machine-learning/concept-workspace?view=azureml-api-2) or [Azure AI project](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/create-projects). Then, you can set the destination via below CLI command:
+
+```console
+pf config set trace.destination=azureml://subscriptions/<subscription-id>/resourcegroups/<resource-group-name>/providers/Microsoft.MachineLearningServices/workspaces/<workspace-or-project-name>
+```
+
+Fill in with your own subscription ID, resource group name, workspace or project name, and all is ready now. You can make LLM calls, run LLM application or execute your flow with `pf flow test` or `pf run create`, you will see an Azure portal URL link in the console:
+
+```console
+You can view the traces in cloud from Azure portal: https://ml.azure.com/prompts/trace/run/<run-name>/details?wsid=/subscriptions/<subscription-id>/resourceGroups/<resource-group-name>/providers/Microsoft.MachineLearningServices/workspaces/<workspace-or-project-name>
+```
+Click the link to view the traces in Azure portal, and feel free to share it with your team members.
```

Comment:
maybe add some screen shot here

Reply:
refer to: https://github.com/Azure/azureai-insiders/blob/chenlu/tracing/previews/tracing-any-genai-app/README.md

Reply:
added one